commercial/app/views/hosted/guardianAmpHostedGalleryImage.scala.html

```diff
@@ -0,0 +1,40 @@
+@import common.commercial.hosted.HostedGalleryImage
+@import views.support.ImgSrc
+@import common.commercial.hosted.HostedGalleryPage
+
+@(picture: model.ImageMedia, image: HostedGalleryImage, index: Int, page: HostedGalleryPage)
+
+@picture.largestImage.map { largestImage =>
```

Comment:
Will this correctly prioritise Grid’s master assets first (it is the same size as the biggest crop, but better quality, but it is solely the master that should go through the resizer)? CAPI has `isMaster` flag to identify master assets.